ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Mempool/Init.hs

```diff
@@ -108,10 +113,11 @@ mkMempool mpEnv =
   Mempool
     { addTx = implAddTx mpEnv
     , removeTxsEvenIfValid = implRemoveTxsEvenIfValid mpEnv
-    , syncWithLedger = implSyncWithLedger mpEnv
     , getSnapshot = snapshotFromIS <$> readTMVar istate
     , getSnapshotFor = implGetSnapshotFor mpEnv
     , getCapacity = isCapacity <$> readTMVar istate
+    , testSyncWithLedger = implSyncWithLedger mpEnv
```

Comment:
```suggestion
    , testSyncWithLedger = implTestSyncWithLedger mpEnv
```
consider renaming the implementation function for consistency. I find it easier to search for this way.